browser/ui/views/tabs/brave_tab.cc

```diff
@@ -15,6 +15,61 @@
 #include "chrome/browser/ui/views/tabs/alert_indicator_button.h"
 #include "chrome/browser/ui/views/tabs/tab_close_button.h"
 #include "chrome/browser/ui/views/tabs/tab_slot_controller.h"
+#include "ui/compositor/layer.h"
+#include "ui/compositor/paint_recorder.h"
+#include "ui/gfx/skia_paint_util.h"
+
+namespace {
+
+#define FILL_SHADOW_LAYER_FOR_DEBUG 0
+
+class ShadowLayer : public ui::Layer, public ui::LayerDelegate {
```

Comment:
Hi, @nullhook ! Just FYI, you might be interested in this patch, which paints a shadow border for a view using ui::Layer.

Reply:
nice!